tests/test_queries.py

```diff
@@ -196,3 +196,58 @@ def test_mapping_resolver_to_object_attribute():
     result = graphql(schema, "{ user { firstName } }")
     assert result.errors is None
     assert result.data == {"user": {"firstName": "Joe"}}
+
+
+def test_default_resolver():
+    type_defs = """
+        type Query {
+            user: User
+        }
+
+        type User {
+            firstName: String
+            avatar(size: String): String
+            blogPosts(published: Boolean): Int
+        }
+    """
+
+    def get_avatar(size):
+        assert size == "200x300"
+        return "test-url"
+
+    def get_blog_posts(published):
+        assert published is True
+        return 3
+
+    resolvers = {
+        "Query": {
+            "user": lambda *_: Mock(
+                first_name=lambda *_: "Joe",
+                avatar=get_avatar,
+                blog_posts=get_blog_posts,
+            )
+        },
+        "User": {
+            "firstName": resolve_to("first_name"),
+            "blogPosts": resolve_to("blog_posts"),
+        },
+    }
+
+    schema = make_executable_schema(type_defs, resolvers)
+
+    query = """
+     query User($size: String, $published: Boolean) {
+      user {
+        firstName
+        avatar(size: $size)
+        blogPosts(published: $published)
+      }
+    }
+    """
+    variables = {"size": "200x300", "published": True}
+
+    result = graphql(schema, query, variables=variables)
+    assert result.errors is None
+    assert result.data == {
+        "user": {"firstName": "Joe", "avatar": "test-url", "blogPosts": 3}
```

Comment:
We might want to reuse those constants in mock and assertion so it would be enough to change one number in future.